api/envoy/extensions/local_address_selectors/filter_state_override/v3/config.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.extensions.local_address_selectors.filter_state_override.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.local_address_selectors.filter_state_override.v3";
+option java_outer_classname = "ConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/local_address_selectors/filter_state_override/v3;filter_state_overridev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Linux Network Namespace Local Address Selector]
```

Comment:
This title should also reflect that it's a "filter state override" local address selector. For now there is only one overridable object, which is the network namespace.